vb-fe/src/core/player/device.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { createContext, useContext, useEffect, useState } from "react";
+import { MUST_BE_WRAPPED_MESSAGE } from "./constant";
+import { useSpotifyPlayerRawInstance } from "./SpotifyPlayer";
+
+export type PlayerDevice = Spotify.WebPlaybackInstance & {
```

Comment:
Is this like a `Type` equivalent of `extends`?

Reply:
nooooo clueeeeeeeeee